lib/install/validate/validator.go

```diff
@@ -222,6 +240,10 @@ func (v *Validator) target(ctx context.Context, input *data.Data, conf *config.V
 			v.NoteIssue(fmt.Errorf("Error processing target after transformation to SOAP endpoint: %q: %s", v.Session.Service, err))
 			return
 		}
+
+		// ESXi requires user/password to be encoded in the Target URL
```

Comment:
As portlayer is gonna change it's behavior, it's good here. Thanks @kjplatz for clarify this.